grc/core/params/dtypes.py

```diff
@@ -13,7 +13,7 @@
 
 
 # Blacklist certain ids, its not complete, but should help
-ID_BLACKLIST = ['self', 'options', 'gr', 'math', 'firdes', 'default'] + dir(builtins)
+ID_BLACKLIST = ['self', 'options', 'gr', 'math', 'firdes', 'default','audio'] + dir(builtins)
```

Comment:
super minor, but would you mind adding the space before `audio` to be consistent?